### PR TITLE
Update history 6 2 2

### PR DESCRIPTION
**Pull request by riversand963**
Update HISTORY.md, and bump version in master to 6.3 since we have released 6.2 and current master should be for 6.3.

### CODE REVIEW

File: include/rocksdb/version.h [hunk 1]
```diff
@@ -5,7 +5,7 @@
 #pragma once
 
 #define ROCKSDB_MAJOR 6
-#define ROCKSDB_MINOR 2
+#define ROCKSDB_MINOR 3
```
Comment (ltamasi) [inline include/rocksdb/version.h]: 6.2.2, right?

Comment (ltamasi) [reply include/rocksdb/version.h]: Oh, never mind, I see you're changing master to 6.3.0.

Comment (riversand963) [reply include/rocksdb/version.h]: I am under the impression that after a major version release, we should bump the version in master to the next major version number.

Comment (riversand963) [reply include/rocksdb/version.h]: Chatted offline, keep this as 6.2.0 for now.

**Pull request closed without merging**